smartcontractkit/smart-contract-db
# feat: adds new entries to frameworks

# Overview

Adds new entries to frameworks

# Screenshots for Mobile and Desktop views (if applicable)

## Before
![image](https://user-images.githubusercontent.com/20880360/137781676-5c28163f-5a2d-4327-9eb5-4f46d277a833.png)

## After
![image](https://user-images.githubusercontent.com/20880360/137781627-bf1530c1-ea27-486d-b482-3d26a5086eba.png)

### Manual Testing

Write the steps required for how might test this. Example:

1. Run `npm run dev`
2. In browser, goto `localhost:3000/`
3. Navigate to Frameworks and verify data
4. Get results :tada:
